puppet/modules/consul/templates/consul.json.epp

```diff
@@ -0,0 +1,7 @@
+<%- | $config | -%>
```

Comment:
can we use an erb file instead and use `<%= require "json"; JSON.pretty_generate(@config) %>`

IMHO this will do the sorting for us as well